### ISSUE TITLE
Add --nbqa-files and --nbqa-exclude flags

### ISSUE THREAD

**Issue by graingert** [nbQA-dev/nbQA#429]
currently nbqa moves files to a tmp dir before formatting - and so flake8's per-file-ignores cannot match files in the repository

**Comment by MarcoGorelli** [comment]
Hi @graingert ,

Thanks for your report!

If you run `nbqa flake8` as a pre-commit hook then you should be able to specify the notebooks/directories you want to exclude, e.g.

```
repos:
  - repo: https://github.com/nbqa-dev/nbqa
    rev: 0.4.0
    hooks:
      - id: nbqa-flake8
        exclude: ^directory/$
```

or, to exclude a file:

```
repos:
  - repo: https://github.com/nbqa-dev/nbqa
    rev: 0.4.0
    hooks:
      - id: nbqa-flake8
        exclude: ^notebook.ipynb$
```

**Comment by MarcoGorelli** [comment]
Having said that, I'd welcome a pull request to add `--nbqa-files` (global file include pattern) and `--nbqa-exclude` flags (global file exclude pattern)

**Comment by MarcoGorelli** [comment]
Hi @graingert 

`--nbqa-files` and `--nbqa-exclude` as available as of version 0.4.1 if you want to try them out, any feedback would be much appreciated! Thanks for the request!
